packages/fast_time/src/lib.rs

```diff
@@ -70,10 +74,14 @@
 //!
 //! # Supported platforms
 //!
-//! This crate is tested on:
+//! This crate provides optimized implementations for:
```

Comment:
The updated text here lies a bit - it says we provide optimized implementaitons for "Ubuntu 24 (Linux)". Actually our optimized implementation also covers other flavors of Linux. The wording is otherwise fine, just remove the "Ubuntu 24" here - that is too specific.